src/containers/applications/wnapp.scss

```diff
@@ -71,20 +76,39 @@ body[data-theme="dark"] .wnstore {
 }
 
 .calcApp {
-  background: hsla(0, 0%, 100%, 0.9);
+  --clr1: rgba(0, 0, 0, 1);
```

Comment:
--clr1 is already a global variable ```--dark-txt: #000;```, use that intead

Reply:
ohh I see  I will replace all the of the colors that can be replaced with global variables :)

Reply:
done :)